lib/core/presentation/widgets/event/event_post_card_widget.dart

```diff
@@ -49,46 +49,40 @@ class EventPostCard extends StatelessWidget {
           ),
         );
       },
-      child: Container(
-        decoration: BoxDecoration(
-          color: colorScheme.surfaceVariant,
-          borderRadius: BorderRadius.circular(LemonRadius.normal),
-        ),
-        child: Stack(
-          children: [
-            Container(
-              decoration: BoxDecoration(
-                border: Border.all(
-                  color: colorScheme.outline,
-                ),
-                borderRadius: BorderRadius.circular(20),
+      child: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              border: Border.all(
```

Comment:
This `Container` widget only required decoration property, consider using `DecoratedBox` for more efficient